exporter/datadogexporter/factory.go

```diff
@@ -285,14 +285,23 @@ func (f *factory) createMetricsExporter(
 
 	ctx, cancel := context.WithCancel(ctx)
 	// cancel() runs on shutdown
+
+	attrsTranslator, err := f.AttributesTranslator(set.TelemetrySettings)
```

Comment:
Ideally we would initialize it just once. Doesn't make much of a difference now, but if it doesn't get addressed on this PR please create a ticket for this

Reply:
I believe it's only initialized once because of the sync.Once variable on the `AttributesTranslator` function?

Reply:
Oh, sorry, misread this :facepalm: you are right